test/project_types/node/commands/generate_test.rb

```diff
@@ -13,7 +13,12 @@ def test_without_arguments_calls_help
         run_cmd('generate')
       end
 
-      def test_for_failure
+      def test_help_argument_calls_extended_help
+        @context.expects(:puts).with { |value| value == extended_help || value == formatted_extended_help }
```

Comment:
I added this or because locally it would be formatted and when running on the CI it wasn't